pipeline/test_general.py

```diff
@@ -0,0 +1,2 @@
+def test_example_test():
+    assert 1 == 1
```

Comment:
I would like to know the purpose of this test? Was this just an example you forgot to remove?